index.html

```diff
@@ -734,7 +734,10 @@ <h3 id="cancelability">Cancelability of touch events</h3>
           <a href="#dfn-canceled-event">Canceling</a> a touch event can prevent or otherwise
           interrupt scrolling (which could be happening in parallel with script execution).
           For maximum scroll performance, a user agent may not wait for each touch event
-          associated with the scroll to be processed to see if it will be canceled.  In such cases
+          associated with the scroll to be processed to see if it will be canceled. In
+          particular, User Agents may not suspend the motion of an element which is currently
```

Comment:
Minor editorial thing: we have been using "user agent"/"user agents" (lowercase) throughout